src/components/calcite-color-hex-input/calcite-color-hex-input.tsx

```diff
@@ -137,9 +137,19 @@ export class CalciteColorHexInput {
   };
 
   private onInputKeyDown = (event: KeyboardEvent): void => {
-    const { inputNode } = this;
-    const { key, altKey, ctrlKey, metaKey } = event;
+    const { key, altKey, ctrlKey, metaKey, shiftKey } = event;
+
+    if (key === "ArrowDown" || key === "ArrowUp") {
```

Comment:
You can leverage the `getKey` util https://github.com/Esri/calcite-components/blob/25e0ba08ace68892b28764bad1bd86fae627f337/src/utils/key.ts#L4 to page over inconsistencies in legacy edge/ie here.

Reply:
💥